### ISSUE TITLE
Add ability to check whether domain is blocked in AGH

### ISSUE THREAD

**Issue by TroyCode3** [AdguardTeam/AdGuardHome#4829]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

#### What problem are you trying to solve?
Everytime when I have to check whether a site is blocked by my existing blocklists, I have to actually visit the site. This is sometimes dangerous as some sites are illegal or may contain malwares.
#### Proposed solution
In DNS blocklists section, add a field to enter the site, and return if any of the blocklists installed block that site (preferrably also show the lists' rule like in query log)

**Comment by ainar-g** [comment]
You mean, like the one that already exists on the *Filters → Custom filtering rules* page?

Improvements to this feature are expected in the v0.108 cycle.

**Comment by TroyCode3** [comment]
Im sorry, but what I meant was to check if a specific list blocks that domain. Currently, the field you mentioned just show only 1 list that filters, but not every list that also filters that site.

**Comment by ainar-g** [comment]
Sounds like a duplicate of #4028 then.
